### PR TITLE
The desktop version

### PR DESCRIPTION
**Pull request by Ayokunnumi1**
These are the features implemented in this branch;
* Media query for min-width 768px,
* New Image files ,
* Grid and flexbox layouts.

I faced a constant Webhint Error on this branch. I tried resolving it but I couldn't. I reached out to my other partners and they told me they faced the same problem and are unable to fix it.

Screenshot 2023-06-10 184954

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -375,6 +420,34 @@ textarea {
...
 
 @media screen and (min-width: 768px) {
+  .header-hero-section {
+    width: 90%;
+    margin: auto;
+    margin-top: 50px;
+    padding: 0 200px;
+    background: url(images/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-size: 100%;
```
Comment (Bantealem) [inline style.css:559]: Please make sure your design is the same as Figma's design. Currently, the back-ground image poison is not the same as the Figma design. Kindly fix it.
Your design.
image

Figma design
image
- [ ] Additionally, please check the horizontal width of the paragraph.

File: style.css [hunk 2]
```diff
@@ -422,5 +511,60 @@ textarea {
...
+    content: url(./images/2nd\ pic.svg);
+  }
+
+  .img-four {
+    content: url(./images/22nd\ pic.svg);
+  }
+
+  .second--child {
+    padding: 24px;
+  }
```
Comment (Bantealem) [inline style.css:657]: Please make sure your design is the same as Figma's design. Currently, the `margin` of your `second--child`  is not the same as the Figma design. Kindly, fix it.
Your design
image

Figma design
image
Your design 
image
Figma design
image
Your design
image
Figma design
image
- [ ] Kindly, decrease the button size too.

File: style.css [hunk 3]
```diff
@@ -386,15 +459,31 @@ textarea {
...
+    margin-top: 5rem;
   }
 
   .about--section {
     width: 100%;
     display: grid;
     grid-template-columns: repeat(2, 1fr);
     gap: 2rem;
-    padding: 20px 100px;
+    padding: 30px 180px 100px 140px;
```
Comment (Bantealem) [inline style.css:600]: Please make sure your design is the same as the given Figma design. Currently the `padding` of your about section is not the same as the one in the Figma. Kindly fix it.
Your design
image
Figma design.
image
- [ ] Kindly, lookt at the paragraph width too.

File: style.css [hunk 4]
```diff
@@ -422,5 +511,60 @@ textarea {
...
+    padding: 16px 143px;
+  }
+
+  .contact--section {
+    background: #6070ff;
+    background-image: url(./images/contact-bkground-icons.png);
+    background-size: 80%;
+    background-repeat: no-repeat;
+    background-position: left 40% top 50%;
+  }
```
Comment (Bantealem) [inline style.css:669]: Please make sure your design is the same as Figma's design. Currently, the background image poison is not the same as the Figma design. Kindly fix it.
Your design
image

Figma design
image

**Review by Bantealem — CHANGES_REQUESTED** [review]
# Status: Required Changes.
Hi @Ayokunnumi1 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
170491251-b1add8ba-b18f-4648-a893-20e510d957fd

## Highlights
- [x] Correct Github flow
- [x] Professional README
- [x] Descriptive PR Summary and title
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -1,267 +1,256 @@
...
+        <ul class="skill--list">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>Javascript</li>
+        </ul>
+        <button type="button" class="see--project">
+          See project
+        </button>
+      </div>
+    </div>
```
Comment (Ofarouq310) [inline index.html:139]: [OPTIONAL]
- [ ] Kindly stick to the `Figma` design specs as much as possible, and try to use the same `paddings` and `margins` as the `Figma` to reach the best look for your project, the same goes for every card on your `Works` section.

| Your design | Figma Design |
| ------------- | ------------- |
| Screenshot 2023-06-11 001250 | image |

File: index.html [hunk 6]
```diff
@@ -1,267 +1,256 @@
...
+      I'd love to help with it!
+    </p>
+    <form action="https://formspree.io/f/xayzglpq" method="Post" class="contact--form">
+      <input type="text" name="name" id="name" required maxlength="30" placeholder="Write your name here">
+      <input type="email" name="email" id="email" required placeholder="Write your email here">
+      <textarea name="message" id="message" maxlength="500" required placeholder="write your message here"></textarea>
+      <button type="submit" class="see--project contact-button">Get in touch</button>
+    </form>
+  </section>
+</body>
```
Comment (Ofarouq310) [inline index.html:254]: [OPTIONAL]
- [ ] Kindly stick to the `Figma` design specs as much as possible, and try to use the same `paddings` and `margins` as the `Figma` to reach the best look for your project.

| Your design | Figma Design |
| ------------- | ------------- |
| Screenshot 2023-06-11 001238 | image |

**Review by Ofarouq310 — APPROVED** [review]
Hi @Ayokunnumi1, ✋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🥇 

giphy

## Highlights
✅ Followed GitHub flow.
✅ Descriptive PR title & summary.
✅ Meaningful commit messages.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**